packages/repo-tools/cli-report.md

```diff
@@ -29,6 +30,15 @@ Options:
   -h, --help
 ```
 
+### `backstage-repo-tools changeset-feedback`
+
+```
+Usage: backstage-repo-tools changeset-feedback [options] <diffRef>
```

Comment:
```suggestion
Usage: backstage-repo-tools changeset-feedback [options] [diffRef]
```
this could be optional if we are assuming `origin/master` or `origin/main` 🤔 
WDYT?

Reply:
Good point